client/jetpack-connect/auth-form-header.jsx

```diff
@@ -120,15 +118,17 @@ class AuthFormHeader extends Component {
 	}
 
 	getSiteCard() {
-		const { jetpackVersion } = this.props;
-		if ( ! versionCompare( jetpackVersion, '4.0.3', '>' ) ) {
+		const { authJpVersion } = this.props;
+		if ( ! versionCompare( authJpVersion, '4.0.3', '>' ) ) {
 			return null;
 		}
 
 		return (
 			<SiteCard
```

Comment:
Might be work quickly checking whether `site-card` needs to exist as a component once we split all the props out of the `queryObject`.